doc/faq.rst

```diff
@@ -58,6 +58,15 @@ fundamentally multi-dimensional. If your data is unstructured or
 one-dimensional, stick with pandas.
 
 
+Gotchas
```

Comment:
Can you frame this as a question, so it looks more similar to the other headings?

e.g., "Why don't aggregations return Python scalars?"